composer.json

```diff
@@ -13,10 +13,7 @@
     "twig/twig": "^2.7.1",
     "wp-cli/wp-cli-bundle": "^2.1@dev",
     "timber/timber": "dev-master",
-    "advanced-custom-fields/advanced-custom-fields-pro": "^5.8.1",
     "wpackagist-plugin/wp-mail-smtp": "^1",
```

Comment:
is it worth removing all plugins now seeing as we'll install them manually and update separately anyway?

Reply:
Good shout, removed those also.